autoload/ale/hover.vim

```diff
@@ -45,7 +45,10 @@ function! ale#hover#HandleTSServerResponse(conn_id, response) abort
             \&& (l:set_balloons is 1 || l:set_balloons is# 'hover')
                 call balloon_show(a:response.body.displayString)
             elseif get(l:options, 'truncated_echo', 0)
-                call ale#cursor#TruncatedEcho(split(a:response.body.displayString, "\n")[0])
+                let l:echo = split(a:response.body.displayString, "\n")
```

Comment:
I think it would be cleaner to check if displayString is empty before calling split:

```
if !empty(a:response.body.displayString)
  call ale#cursor#TruncatedEcho(split(a:response.body.displayString, "\n")[0])
fi
```

Reply:
Didn't realize that API existed, nice, thanks!